dispatch/resources/messages.py

```diff
@@ -0,0 +1,18 @@
+import dispatch
+
+DISPATCH_ADMIN_HELP_TEXT = """
+Dispatch Admin """+ dispatch.__version__ + """
+
+Usage: 
+    dispatch-admin <OPTION> <ARGUMENT>
+
+ Options:
+    test            Run tests
```

Comment:
Check back on the docopt specification here. Looks like they [expect all options to start with - or --](https://github.com/docopt/docopt#option-descriptions-format)